packages/core/src/browser/style/search-box.css

```diff
@@ -24,7 +32,7 @@
     box-shadow: var(--theia-border-width) var(--theia-border-width) var(--theia-widget-shadow);
     line-height: var(--theia-private-horizontal-tab-height);
     background-color: var(--theia-editor-background);
-    z-index: 1;
+    z-index: var(--theia-toolbar-z-index);
```

Comment:
It should be call against some other variable? How do you get 1110? To which value add which to be such number? It should be a formula expressing it.

Reply:
@akosyakov  Since I couldn't find a css variable for toolbar z-index, I created a new variable. 

Currently, there is only one css variable for z-index called "--theia-menu-z-index". Should I use that variable instead?

Reply:
@akosyakov perhaps we should do the refactoring to use variables at a later point and revisit all instances of using a hardcoded `z-index`. Ideally all these references should use variables and their `z-index` be computed as to avoid issues. We currently have a lot of violations of such an approach:

<img width="533" alt="Screen Shot 2020-02-10 at 5 08 11 PM" src="https://user-images.githubusercontent.com/40359487/74194858-18669400-4c28-11ea-8823-692edd0c7872.png">

Reply:
Could you explain please for this z-index how to you get 1110? Should not we add a variable for z-index of the underlying toolbar items and here just use calc from this variable, i.e. `calc(var(--theia-tab-z-index) + 1)`?

Reply:
> perhaps we should do the refactoring to use variables at a later point and revisit all instances of using a hardcoded z-index.

revisiting all of them is another issue, we can do it, but i don't think it is in the scope of this PR

Reply:
> > perhaps we should do the refactoring to use variables at a later point and revisit all instances of using a hardcoded z-index.
> 
> revisiting all of them is another issue, we can do it, but i don't think it is in the scope of this PR

Yes, that's what I mentioned, we should revisit these violations in order to calculate the `z-index` in a smarter way in a subsequent pull-request. I think what you mentioned previously about calculating the value would be fine for me.